### PR TITLE
FNI-4199 - docs: deleted a bunch of stuff

### PR DESCRIPTION
**Pull request by grahamlangford**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -5,20 +5,6 @@
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
 
-                            Preamble
-
```
Comment (bens5018) [inline LICENSE:9]: trash change!

File: LICENSE [hunk 2]
```diff
@@ -5,20 +5,6 @@
...
-  The GNU General Public License is a free, copyleft license for
-software and other kinds of works.
-
-  The licenses for most software and other practical works are designed
-to take away your freedom to share and change the works.  By contrast,
-the GNU General Public License is intended to guarantee your freedom to
-share and change all versions of a program--to make sure it remains free
-software for all its users.  We, the Free Software Foundation, use the
-GNU General Public License for most of our software; it applies also to
-any other work released this way by its authors.  You can apply it to
```
Comment (cdeckert-fni) [inline LICENSE:19]: I like what you did here

File: LICENSE [hunk 3]
```diff
@@ -5,20 +5,6 @@
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
 
-                            Preamble
```
Comment (rjensen-sketch) [inline LICENSE:8]: I guess this was a good idea

File: LICENSE [hunk 4]
```diff
@@ -5,20 +5,6 @@
...
-
-  The licenses for most software and other practical works are designed
-to take away your freedom to share and change the works.  By contrast,
-the GNU General Public License is intended to guarantee your freedom to
-share and change all versions of a program--to make sure it remains free
-software for all its users.  We, the Free Software Foundation, use the
-GNU General Public License for most of our software; it applies also to
-any other work released this way by its authors.  You can apply it to
-your programs, too.
-
```
Comment (rjensen-sketch) [inline LICENSE:21]: WHY DID YOU DELETE THIS BRO?

**Review by bens5018 — APPROVED** [review]

File: LICENSE [hunk 5]
```diff
@@ -5,20 +5,6 @@
...
   When we speak of free software, we are referring to freedom, not
 price.  Our General Public Licenses are designed to make sure that you
```
Comment (rjensen-sketch) [inline LICENSE:9]:
```suggestion
price.  Our General Public Licenses are designed to make sure that you!!!!!!!!!!!!!!!!
```

Comment (cdeckert-fni) [inline LICENSE:9]:
```suggestion
price.  Our General Public Licenses are designed to make sure that you do something awesome!!
```

**Review by rjensen-sketch — CHANGES_REQUESTED** [review]
not too shabby

**Review by cdeckert-fni — CHANGES_REQUESTED** [review]
See suggested changes

**Review by jallen-fni — COMMENTED** [review]
this is horrible why would you do this.

**Review by jallen-fni — APPROVED** [review]
much better

**Review by jallen-fni — APPROVED** [review]

**Comment by Brandons404** [comment]
This is no longer the best practice way to do this.

File: LICENSE [hunk 6]
```diff
@@ -5,20 +5,6 @@
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
```
Comment (melmir-eng) [inline LICENSE:6]:
```suggestion
 of this license document, but changing it is not allowed.
```

**Review by bens5018 — APPROVED** [review]
